### DISCUSSION TITLE
GeolocateControl not appearing

### DISCUSSION THREAD

**Discussion by thanosoncode** [uber/react-map-gl#2371] [General]
```
"react-map-gl": "^7.1.7",
 "mapbox-gl": "^3.2.0",
 "react": "18.2.0",
```
What else do I need to add in order GeolocateControl to appear? I use the example from the [docs](https://visgl.github.io/react-map-gl/docs/api-reference/geolocate-control)

```ts
import Map, { GeolocateControl } from 'react-map-gl';

  <Map
      mapboxAccessToken={process.env.MAPBOX_ACCESS_TOKEN}
      initialViewState={{
        longitude: -100,
        latitude: 40,
        zoom: 3.5,
      }}
      mapStyle='mapbox://styles/mapbox/streets-v9'
    >
      <GeolocateControl position='top-right' />
    </Map>
```
![Screenshot 2024-04-09 at 7 44 49 AM](https://github.com/visgl/react-map-gl/assets/74588018/ad07d7ae-9c7b-45f9-83c2-01098ee37633)

When inspecting I see a `button class="mapboxgl-ctrl-geolocate" however it is outside of the map and not visible. How can I manipulate this button in order to make it appear?

**Comment by Pessimistress** [comment]
https://visgl.github.io/react-map-gl/docs/get-started#styling

**Reply by thanosoncode** [reply]
thanks mate!
